core/src/main/java/com/sequenceiq/cloudbreak/converter/v4/stacks/StackV4RequestToStackConverter.java

```diff
@@ -323,14 +318,15 @@ private void validateStackAuthentication(StackV4Request source) {
     private void updateCluster(StackV4Request source, Stack stack, Workspace workspace) {
         if (source.getCluster() != null) {
             source.getCluster().setName(stack.getName());
-            Cluster cluster = getConversionService().convert(source.getCluster(), Cluster.class);
-            Set<HostGroup> hostGroups = source.getInstanceGroups().stream()
+            Cluster cluster = measure(() -> getConversionService().convert(source.getCluster(), Cluster.class),
```

Comment:
measure() leads to unreadable code, I am not sure why this is preferred instead of annotating a method